pyproject.toml

```diff
@@ -43,7 +43,7 @@ pynumaflow = "~0.5"
 mlflow-skinny = { version = "^2.0", optional = true }
 redis = {extras = ["hiredis"], version = "^5.0", optional = true}
 boto3 = { version = "^1.24.64", optional = true }
-pydruid= {git="https://github.com/cosmic-chichu/pydruid.git", branch = "master", optional = true }
+pydruid= {git="https://github.com/cosmic-chichu/pydruid.git", branch = "addDoublesSketch", optional = true }
```

Comment:
Can you push to your master branch instead?

Reply:
Updating master will change the upstream PR for intervals isn't it?